src/behaviors/clap/clap.js

```diff
@@ -1,28 +1,28 @@
 import Behavior from '../behavior.js';
 
-class RollTheDice extends Behavior {
+class ClaporRave extends Behavior {
   constructor(settings) {
-    settings.name = 'Clap Hype';
-    settings.description = `'cause 👏 sometimes 👏 you 👏 need 👏 emphasis!`;
+    settings.name = 'ClappingRave';
+    settings.description = `'cause sometimes you need emphasis!`;
     super(settings);
 
-    this.commands.push({
-      tag: 'clap',
-      description: `I'll hype your message for you 👏👏👏`
-    });
+    ['clap', 'rubyrave', 'wut'].forEach(emoji => this.commands.push({
+      tag: emoji,
+      description: `I'll hype your message, !Clap or !Rave`
```

Comment:
Will `!Clap` work, or only `!clap`? You should make sure what users see is what they can use. I've tried to strive for all lower case for these where possible.

Reply:
Agreed, Pesky Shift Trigger Finger. 

Will lowercase